spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -3882,14 +3908,14 @@ def _handle_completions(self):
                     self._last_key_pressed_text = ''
                     self._last_pressed_key = None
 
-    def fix_and_strip_indent(self, comment_or_string=False):
+    def fix_and_strip_indent(self, *args, **kwargs):

```

Comment:
Maybe we could specify the `kwargs` description in the docstring to be clear about the possible `kwargs` the method receives?